org.ardour.Ardour.json

```diff
@@ -0,0 +1,412 @@
+          "type": "archive",
+          "url": "https://download.drobilla.net/serd-0.30.0.tar.bz2",
+          "sha256": "6efb0efa5c2155e6bbac941cddeeabb7ed26d70a57d24178894ff169d8f6cefb"
+        }
+      ],
+      "post-install": [
+        "install -Dm644 -t /app/share/licenses/serd COPYING"
+      ],
+      "cleanup": [
+        "/bin",
+        "/share/man"
+      ]
+    },
+    {
+      "name": "sord",
+      "buildsystem": "simple",
+      "build-commands": [
+        "python3 ./waf configure --prefix=/app",
+        "python3 ./waf build -j $FLATPAK_BUILDER_N_JOBS",
+        "python3 ./waf install"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.drobilla.net/sord-0.16.2.tar.bz2",
+          "sha256": "09f51174dd8f3efbd95f44f0bb0b165f08e066e052d40095de59de787987da8d"
+        }
+      ],
+      "post-install": [
+        "install -Dm644 -t /app/share/licenses/sord COPYING"
+      ],
+      "cleanup": [
+        "/bin",
+        "/share/man"
+      ]
+    },
+    {
+      "name": "sratom",
+      "buildsystem": "simple",
+      "build-commands": [
+        "python3 ./waf configure --prefix=/app",
+        "python3 ./waf build -j $FLATPAK_BUILDER_N_JOBS",
+        "python3 ./waf install"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.drobilla.net/sratom-0.6.2.tar.bz2",
+          "sha256": "0a514a55d6b6cb7b5d6f32d1dcb78a1e6e54537fa22fce533e4ef6adf240e853"
+        }
+      ],
+      "post-install": [
+        "install -Dm644 -t /app/share/licenses/sratom COPYING"
+      ]
+    },
+    {
+      "name": "lilv",
+      "buildsystem": "simple",
+      "build-commands": [
+        "python3 ./waf configure --prefix=/app",
+        "python3 ./waf build -j $FLATPAK_BUILDER_N_JOBS",
+        "python3 ./waf install"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.drobilla.net/lilv-0.24.4.tar.bz2",
+          "sha256": "c33b84b7a6e8e8fffb412fbcd6f69e59ca297ef3e29d829249b4ccc94f634438"
+        }
+      ],
+      "post-install": [
+        "install -Dm644 -t /app/share/licenses/lilv COPYING"
+      ],
+      "cleanup": [
+        "/bin",
+        "/etc",
+        "/share/man"
+      ]
+    },
+    {
+      "name": "ardour",
+      "buildsystem": "simple",
+      "build-commands": [
+        "./waf configure --prefix=/app --no-phone-home --freedesktop --with-backends=jack,alsa,dummy",
+        "./waf build -j $FLATPAK_BUILDER_N_JOBS",
+        "./waf i18n -j $FLATPAK_BUILDER_N_JOBS",
+        "./waf install"
+      ],
+      "post-install": [
+        "desktop-file-edit --set-key=Name --set-value=\"Ardour 5\" --set-generic-name=\"Digital Audio Workstation\" --set-key=X-GNOME-FullName --set-value=\"Ardour v5 (Digital Audio Workstation)\" --set-comment=\"Record, mix and master audio\" --remove-category=AudioEditing --add-category=X-AudioEditing build/gtk2_ardour/ardour5.desktop",
```

Comment:
Why you are doing it here? I mean, you have already provided the whole `ardour5.desktop` file, so this is questionable.

In my opinion, you should either:
1. Just provide modified `ardour.desktop.in` file in sources (just remember about `dest`: `gtk2_ardour`) and don't use the `desktop-file-edit` command here.
2. Don't provide `ardour.desktop.in` nor `ardour5.desktop` file, and just edit auto-generated `build/gtk2_ardour/ardour5.desktop` file in post-install section using `desktop-file-edit` command.

Reply:
Upon reflection, I would just add `-set-icon=${FLATPAK_ID}` here:
```
        "desktop-file-edit --set-key=Name --set-value=\"Ardour 5\" --set-generic-name=\"Digital Audio Workstation\" --set-key=X-GNOME-FullName --set-value=\"Ardour v5 (Digital Audio Workstation)\" --set-comment=\"Record, mix and master audio\" --remove-category=AudioEditing --add-category=X-AudioEditing --set-icon=${FLATPAK_ID} build/gtk2_ardour/ardour5.desktop",
```

Reply:
Of course you still have to install the AppData file manually, as I mentioned earlier.
https://github.com/flathub/flathub/pull/831#issuecomment-461332534

Reply:
done